_includes/parse-server/deploying-aws-ubuntu.md

```diff
@@ -33,17 +33,14 @@ Check the install was ok, you should see the version installed.
 node -v
 ```
 
-
 Install [npm](https://www.npmjs.com)
 ```bash
 sudo apt install npm
 ```
 
-
-
 Install [yarn](https://yarnpkg.com)
 ```bash
-sudo npm install yarn –g
+sudo npm install yarn -g
```

Comment:
Why is yarn required in this example? I don't see any yarn command being executed?